languoids/tree/atla1278/nort3146/tend1239/bass1263/bedi1237/bape1239/md.ini

```diff
@@ -28,7 +28,7 @@ glottolog =
 [hh_ethnologue_comment]
 comment = The Bapen language is (or was) not intelligible to its closest relatives Bassari [bsc] and Bedik [tnr]
 	(**hh:w:SantosFerry:Wey-Pe**:82,
-	**hh:vld:Ferry:Bapen**:353-354). It was nearly extinct in 1971.  I wish to thank Guillaume Segerer for bringing this language to my attention. It is missing from E16/E17/E18/E19/E20/E21/E22/E23. See also: Bedik [tnr], Bassari-Tanda [bsc].
+	**hh:vld:Ferry:Bapen**:353-354). It was nearly extinct in 1971.  I wish to thank Guillaume Segerer for bringing this language to my attention. It is missing from E16/E17/E18/E19/E20/E21/E22/E23. See also: Bassari-Tanda [bsc], Bedik [tnr].
```

Comment:
@d97hah it looks like you should explicitly sort the languages listed after "See also". Otherwise we might get these apparent changes, which makes for confusing histories.